src/sentry/static/sentry/app/components/organizations/headerSeparator.jsx

```diff
@@ -3,7 +3,7 @@ import {Box} from 'grid-emotion';
 
 const HeaderSeparator = styled(Box)`
   width: 1px;
-  background-color: ${p => p.theme.offWhite};
+  background-color: ${p => p.theme.borderLight};
   margin: 4px 16px;
```

Comment:
Not related to your changes but I'd prefer if we drop the margin from here and everything inside the boxed area forms part of the projects and time selectors, otherwise we're defining the other parts of this margin in all different components, and the clickable area doesn't extend to the border as one might expect

Reply:
As of now, the dropdown clickable area extends from the start of the text to the caret -- which I don't think is unusual.
The margin makes a small non-clickable area but i think it makes sense that the space in between isn't clickable imo.